pilot/pkg/networking/core/tracing_test.go

```diff
@@ -546,6 +547,28 @@ func defaultTracingTags() []*tracing.CustomTag {
 				},
 			},
 		},
+		&tracing.CustomTag{
+			Tag: "istio.destination_workload",
```

Comment:
`destination_workload` already has some semantic meaning; it's usually just the workload name. Can we make this `destination_workload_metadata` or _attributes or something?